examples/step-19/step-19.cc

```diff
@@ -650,6 +651,9 @@ namespace Step19
   {
     const double dt = time.get_next_step_size();
 
+    std::vector<double>         solution_values(fe.n_dofs_per_cell());
+    std::vector<double>         field_values;
```

Comment:
We don't actually need `field_values` at all. Maybe call it `dummy_field_values` and not resize it at all?

Reply:
@kronbichler and I had a short discussion on how to improve the interface of the new class (see https://github.com/dealii/dealii/pull/11046#issuecomment-753946564); as a consequence these lines will changes. By the time of merging of the PR, we knew that the interface was was not final but we intended to improve it after we made some experiences during usage in actual application codes. I hope I have time tomorrow to work on the redesign of the interface.